SPECS/mstflint/mstflint.spec

```diff
@@ -23,15 +23,35 @@ BuildRequires:  libxml2-devel
 BuildRequires:  make
 BuildRequires:  openssl-devel
 BuildRequires:  zlib-devel
+BuildRequires:  kernel-devel
+BuildRequires:  kmod
 Requires:       python3
+Requires:       kernel
+
+# set package name
+%{!?_name: %global _name mstflint}
+
+%global kver %(/bin/rpm -q --queryformat '%{RPMTAG_VERSION}-%{RPMTAG_RELEASE}' $(/bin/rpm -q --whatprovides kernel-devel))
+%global ksrc %{_libdir}/modules/%{kver}/build
+%global debug_package %{nil}
```

Comment:
Removing debuginfo packages is a global setting for the whole spec, so it will also remove any debug packages for the other subpackages. This is a regression. Why do we want to remove the debuginfo packages?